src/ansys/sherlock/core/analysis.py

```diff
@@ -708,21 +722,23 @@ def update_ict_analysis_props(
             raise e
 
     @require_version(241)
-    def get_mechanical_shock_input_fields(self, model_source=None):
+    def get_mechanical_shock_input_fields(
+        self, model_source: Optional[ModelSource] = None
+    ) -> list[str]:
         """Get mechanical shock property fields based on the user configuration.
 
         Available Since: 2024R1
 
         Parameters
         ----------
-        model_source : ModelSource, optional
+        model_source: ModelSource, optional
             Model source to get the random vibe property fields from.
-            Only ModelSource.GENERATED is supported.
+            Only GENERATED is supported.

```

Comment:
I removed `ModelSource.` here because it seems redundant to specify the type here. I don't mind reverting this.